### ISSUE TITLE
detected bounding box slightly rotated to one direction

### ISSUE THREAD

**Issue by handsomeRobot** [dengdan/seglink#50]
Hi, dengdan, thanks for your repo to help me with the process of my project. I am now training on my own dataset, with your pre-trained 512 weights. Everything goes fine, but I met a problem when doing inference with my trained weights. The bboxes found by my weights are all slightly rotated to a particular orientation. More specifically, all horizontal bboxes are slightly rotated clockwise to various degrees, like 5 degree to 15 degree. This is wired since no bboxes are rotated anti-clockwise. Just before I check the code with care, could you give me a hint of what might be going wrong? Thanks in advance.

**Comment by handsomeRobot** [comment]
Some changes I do:
1. xrange -> range to be compatible with my python3 env.
2. I used opencv3, no error encounted, do not know if this is the issue.
3. I replaced optimizer from Momentum to Adam to bypass NaN loss error.
4. My own data is PASCAL format, so I rewrite a script to write it into tf-records, the data looks good in tensorboard.
